src/web/lib/components/BrowserPreview/index.scss

```diff
@@ -8,10 +8,9 @@
 
 .doll--small {
   @include unit('small');
-  width: $grid * 50;
   height: $grid * 13.25;
 
-  .doll__tabbar {
+  > .doll__tabbar {
```

Comment:
Added some direct descendant selectors here in case we want to nest these things